src/site/_includes/components/Instruction.js

```diff
@@ -101,21 +101,15 @@ module.exports = (type, listStyle = 'ul') => {
       break;
 
     case 'disable-cache':
-      instruction = html`
-        ${bullet}Select the **Disable cache** checkbox.
-      `;
+      instruction = html` ${bullet}Select the **Disable cache** checkbox. `;
```

Comment:
probably not a huge deal but maybe we should delete the whitespace at the start and end of these strings.